include/ringmesh/mesh/cartesian_grid.h

```diff
@@ -281,11 +281,160 @@ namespace RINGMesh
          * @return the attribute manager containing the grid point attributes
          * values.
          */
-        GEO::AttributesManager& attributes_manager() const
+        GEO::AttributesManager& attributes_manager()
         {
             return attributes_manager_;
         }
 
+        /*!
+         * @return the attribute manager containing the grid point attributes
+         * values.
+         */
+        const GEO::AttributesManager& attributes_manager() const
+        {
+            return attributes_manager_;
+        }
+
+        void add_index_t_attribute( const std::string& attribute_name,
```

Comment:
why don't create add_attribute<T>?

Reply:
I tried doing a template function for adding attributes but it didn't work (I had difficulty transforming a templated type into a string to tell what type of attribute to add)
I will try again when I have more time, but these 2 functions allow the implementation in different algorithms in my RINGGrid plugin.

Reply:
It would be great if this function was template.